packages/extension/src/ui/features/accountTokens/HideTokenScreen.tsx

```diff
@@ -53,7 +53,7 @@ export const HideTokenScreen: FC = () => {
 
   const handleSubmit = () => {
     try {
-      removeToken(token)
+      hideToken(token)
```

Comment:
It feels that `removeToken` is more appropriate to Argent (or other EVM based chain), since the tokens are not first class citizen and if it is removed from the token store, then its definition is removed. 
For us, since tokens are first class citizen, we always get them from the UTXOs and store them if they do not exist in the token store, so maybe "hide" makes more sense for us. 
WDYT?

Reply:
Is `removeToken` removing the token from the chain or just removing the metadata from localStoraage?

Reply:
It's removed from the local storage.

Reply:
Agreed that `Hide` sounds better since we don't remove the token metadata from the storage